binding/react-native/android/src/main/java/ai/picovoice/reactnative/porcupine/PorcupineModule.java

```diff
@@ -44,28 +44,48 @@ public String getName() {
     return "PvPorcupine";
   }
 
-   @Override
-  public Map<String, Object> getConstants() {
-    // default model file
-    final File resourceDirectory = reactContext.getFilesDir();
-    final Map<String, Object> constants = new HashMap<>();
-    constants.put("DEFAULT_MODEL_PATH", new File(resourceDirectory, "porcupine_params.pv").getAbsolutePath());
-
-    // default keyword files
-    final Map<String, String> keywordPaths = new HashMap<>();
-    for (Porcupine.BuiltInKeyword x : Porcupine.BuiltInKeyword.values()) {
-      String fileName = x.name().toLowerCase();
-      String keyword = fileName.replace('_', ' ');
-      keywordPaths.put(keyword, new File(resourceDirectory, fileName + ".ppn").getAbsolutePath());
+  @ReactMethod
+  public void fromBuiltInKeywords(String accessKey, String modelPath, ReadableArray keywords, ReadableArray sensitivities, Promise promise) {
```

Comment:
I think there needs to be more shared code between `fromBuiltInKeywords` and `fromKeywordPaths`. Probably best to do it like we do in other bindings and have the constructor take keyword paths and `fromBuiltInKeywords` just converts to paths and passes into that constructor.